pkg/build/api/types.go

```diff
@@ -328,6 +328,9 @@ type GitBuildSource struct {
 
 	// HTTPSProxy is a proxy used to reach the git repository over https
 	HTTPSProxy *string
+
+	// URLCheckTimeoutSeconds is the timeout in seconds to use when checking connectivity to the git server
+	URLCheckTimeoutSeconds *int
```

Comment:
We should never be using Duration in an API.

On Mon, May 23, 2016 at 3:37 PM, David Eads <notifications@github.com>
wrote:

> In pkg/build/api/types.go
> <https://github.com/openshift/origin/pull/8990#discussion_r64275012>:
>
> > @@ -328,6 +328,9 @@ type GitBuildSource struct {
> >
> >  	// HTTPSProxy is a proxy used to reach the git repository over https
> >  	HTTPSProxy *string
> > +
> > +	// URLCheckTimeoutSeconds is the timeout in seconds to use when checking connectivity to the git server
> > +	URLCheckTimeoutSeconds *int
>
> We've used time.Duration elsewhere (even in BuildStatus), why use an int
> here?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8990/files/09988784f42909e7b7cd9619c04c27904d62d89d#r64275012>
>